advanced/helm/integrator-with-analytics/integrator-with-analytics/templates/integrator-analytics-deployment.yaml

```diff
@@ -12,31 +12,56 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: integrator-with-analytics-ei-analytics-deployment
-  namespace : {{ .Values.namespace }}
+  namespace : {{ .Release.Namespace }}
 spec:
-  replicas: 2
-  minReadySeconds: 30
+  replicas: {{ default 2 .Values.wso2.deployment.wso2eiAnalyticsWorker.replicas }}
+  minReadySeconds: {{ default 2 .Values.wso2.deployment.wso2eiAnalyticsWorker.minReadySeconds }}
   strategy:
     rollingUpdate:
-      maxSurge: 1
-      maxUnavailable: 0
+      maxSurge: {{ default 1 .Values.wso2.deployment.wso2eiAnalyticsWorker.strategy.rollingUpdate.maxSurge }}
+      maxUnavailable: {{ default 0 .Values.wso2.deployment.wso2eiAnalyticsWorker.strategy.rollingUpdate.maxUnavailable }}
     type: RollingUpdate
+  selector:
+    matchLabels:
+      deployment: integrator-with-analytics-ei-analytics
   template:
     metadata:
       labels:
         deployment: integrator-with-analytics-ei-analytics
     spec:
+      initContainers:
+      - name: init-is-db
+        image: busybox:1.31
+        command: ['sh', '-c', 'echo -e "Checking for the availability of MySQL Server deployment"; while ! nc -z {{ default "wso2ei-rdbms-service-mysql" .Values.wso2.mysql.host }} 3306; do sleep 1; printf "-"; done; echo -e "  >> MySQL Server has started";']
+      {{ if .Values.wso2.centralizedLogging.enabled }}
+      - name: init-logstash-plugins-install
+        image: docker.elastic.co/logstash/logstash:{{ default "7.2.0" .Values.wso2.centralizedLogging.logstash.imageTag }}
+        command:
+          - /bin/bash
```

Comment:
@Savidude this has to be changed Shell (`/bin/sh`) rather than Bash (`/bin/bash`). Bash is not present in Alpine based Docker images by default. Hence, this probe command may fail. This is applicable to all WSO2 product profiles.

Reply:
Fixed https://github.com/wso2/kubernetes-ei/pull/180/commits/4eab0894505a022e409f632e44196a919575ceb1